src/System.Text.Encoding/tests/Decoder/Decoder.cs

```diff
@@ -223,6 +223,38 @@ public static void DecoderExceptionFallbackBufferTest()
             Assert.Equal('\u0000', fallbackBuffer.GetNextChar());
         }
 
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework contains known bugs in this code path.")]
```

Comment:
@ViktorHofer, based on your previous comments, this should not be added?

Reply:
If the test project doesn't have a netfx configuration then you don't need that line.

Reply:
The test project only has _netcoreapp_ and _uap_ configurations. I'll remove the line.